### PR TITLE
promtail: Add version 1.3.0

### PR DESCRIPTION
**Pull request by bbigras**

### CODE REVIEW

File: bucket/promtail.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "checkver": {
+    "github": "https://github.com/grafana/loki"
+  },
+  "autoupdate": {
+    "architecture": {
+      "64bit": {
+        "url": "https://github.com/grafana/loki/releases/download/v$version/promtail-windows-amd64.exe.zip"
+      }
+    }
```
Comment (Ash258) [inline bucket/promtail.json]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/SHA256SUMS"
        }
```

File: bucket/promtail.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "version": "1.3.0",
+  "description": "log agent for Loki",
```
Comment (Ash258) [inline bucket/promtail.json]:
```suggestion
    "description": "Multi-tenant log aggregation system",
```

File: bucket/promtail.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "homepage": "https://grafana.com/loki",
```
Comment (Ash258) [inline bucket/promtail.json]:
```suggestion
    "homepage": "https://grafana.com/oss/loki",
```

File: bucket/promtail.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  "license": "Apache-2.0",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/grafana/loki/releases/download/v1.3.0/promtail-windows-amd64.exe.zip",
+      "hash": "988e1c314101a5afbd4dd322e396dff3fc7e2feece1cb5c299c70321f1135ce5"
```
Comment (Ash258) [inline bucket/promtail.json]:
```suggestion
            "hash": "988e1c314101a5afbd4dd322e396dff3fc7e2feece1cb5c299c70321f1135ce5",
            "bin": [
                [
                    "promtail-windows-amd64.exe",
                    "promtail"
                ]
            ]```

File: bucket/promtail.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  },
+  "bin": [
```
Comment (Ash258) [inline bucket/promtail.json]:
```suggestion
```

Comment (bbigras) [reply bucket/promtail.json]: `bin` is not needed?

Comment (Ash258) [reply bucket/promtail.json]: https://github.com/lukesampson/scoop-extras/pull/3527#discussion_r370918023 ...

**Comment by bbigras** [comment]
I guess I need to remove the ``` too.

After resolving the `bin` thing I should make a new PR in the main bucket right?

**Comment by Ash258** [comment]
There is no "bin thing" to resolve. Open PR into Main bucket

**Comment by bbigras** [comment]
OK thanks. Sorry for the confusion.

**Pull request closed without merging**